src/modules/ROOT/pages/changelog/202309/00.json

```diff
@@ -0,0 +1,261 @@
+      ]
+    },
+    {
+      "what": [
+        "a rounded circle icon that marks a configuration option as performance impacting."
+      ],
+      "why": [
+        "to mark which features would impact the performance of an application."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1490"
+      ]
+    }
+  ],
+
+  "changed": [
+    {
+      "what": [
+        "triggering of unsaved changes message after user wants to delete a modified resource like a page, namespace, module or chart."
+      ],
+      "why": [
+        "to provide a more intuitive way to navigate in {APP_NAME_COMPOSE} after trying to delete a resource."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1466"
+      ]
+    },
+    {
+      "what": [
+        "the navigation flow when switching between different {APP_NAME_COMPOSE} resources using the sidebar navigation menu.",
+        "For example, when trying to switch from one page's builder mode to another, you were redirected to the edit mode of the page."
+      ],
+      "why": [
+        "to provide a more intuitive way to navigate between {APP_NAME_COMPOSE} resources."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1457"
+      ]
+    },
+    {
+      "what": [
+        "the functionality of the delete and un-delete button by extending it to show a spinner while a resource is being deleted or un-deleted."
+      ],
+      "why": [
+        "to show the user the current state of the delete or un-delete process."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1464"
+      ]
+    },
+    {
+      "what": [
+        "the syntax of forms for creating and modifying a resource to better support mobile users in {APP_NAME_ADMIN}."
+      ],
+      "why": [
+        "to provide a better UI and UE for mobile users."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1253"
+      ]
+    },
+    {
+      "what": [
+        "the updating cycle of page blocks showing the same data to refresh after a change has been made to that data in a different page block."
+      ],
+      "why": [
+        "to keep all page blocks showing related data in sync."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1435"
+      ]
+    },
+    {
+      "what": [
+        "general ui/ux improvements:",
+        "** Improved header button layout for topbar navigation and resource list items.",
+        "** Adjusted position of btn:[x] in search bar to always appear on the right side."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1489",
+        "https://github.com/cortezaproject/corteza/commit/dcc541d953d61172797117cddc929d327023fa0a"
+      ]
+    }
+  ],
+
+  "fixed": [
+    {
+      "what": [
+        "the following mobile issues in {APP_NAME_ADMIN}:",
```

Comment:
I'd personally move this to general UI/UX changes; some mobile stuff -- not sure we need to have a whole entry for this.

If you wish, you can keep it but change the `-` to `**` as above